vendor_skins/macUnskin.sh

```diff
@@ -1,5 +1,15 @@
+rm CI/install/osx/Info.plist
+rm CI/install/osx/dylibBundler
+rm CI/install/osx/packageApp.sh
 git checkout -- CI/*
 rm UI/webrtcVersion.h
 git checkout -- UI/*
 git checkout -- cmake/*
-
+git checkout -- deps/*
+rm libobs/obs-cocoa.m
+git checkout -- libobs/*
+rm libobs-d3d11/d3d11-texture3d.cpp
+git checkout -- libobs-d3d11/*
+rm libobs-opengl/gl-texture3d.c
+git checkout -- libobs-opengl/*
```

Comment:
What is this file for, can you tell me?  "MacUnskin" sounds weird

Reply:
Aha, this is actually important.  When the project is being built, we must copy files from the vendor_skins/Evercast dir into the root.  That messes up git big-time.  Therefore there is a convenient little script here to restore the natural order of things.  I have extended it somewhat, since we have ported a large chunk of OBS into EBS.

Reply:
Nice, thanks for your effort in maintaining compatibility @SteveWestbrook.